CHANGELOG.md

```diff
@@ -14,6 +14,15 @@ Thank you @Eneuman,@Tatsinnit, @BartNetJS and @cxznmhdcxz for great contribution
 - [Update support doc with correct links](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/69)
 - [Change to service.name to show correct ingress url's](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/72)
 
+## [2.0.120231128]
```

Comment:
🐛  Why is this not at the top of this `changelog`? I think auto release will not detect this. 🥷❤️ cc: @hsubramanianaks  and @Vidya2606

Reply:
good one @Tatsinnit we missed it.